src/Documentation/grains/transactions.md

```diff
@@ -88,31 +116,54 @@ public class AccountGrain : Grain, IAccountGrain
     private readonly ITransactionalState<Balance> balance;
 
     public AccountGrain(
-        [TransactionalState("balance")] ITransactionalState<Balance> balance)
+        [TransactionalState("balance", "TransactionStore")]
+        ITransactionalState<Balance> balance)
     {
         this.balance = balance ?? throw new ArgumentNullException(nameof(balance));
     }
 
     Task IAccountGrain.Deposit(uint ammount)
     {
-        this.balance.State.Value += ammount;
-        this.balance.Save();
-        return Task.CompletedTask;
+        return this.balance.PerformUpdate(x => x.Value += amount);
     }
 
     Task IAccountGrain.Withdrawal(uint ammount)
     {
-        this.balance.State.Value -= ammount;
-        this.balance.Save();
-        return Task.CompletedTask;
+        return this.balance.PerformUpdate(x => x.Value -= amount);
     }
 
     Task<uint> IAccountGrain.GetBalance()
     {
-        return Task.FromResult(this.balance.State.Value);
+        return this.balance.PerformRead(x => x.Value);
     }
-}
+}`
+```
+
+In the above example, the attribute `TransactionalState` is used to declare that the ‘balance’ constructor argument should be associated with a transactional state named “balance”.
+With this declaration, Orleans will wire up an `ITransactionalState` instance with a state loaded from the transactional state storage named "TransactionStore" (see Setup).
```

Comment:
Maybe "inject" instead of "wire up", but I don't hold a strong opinion

Reply:
I made the change. When @jason-bragg sends out the related "Setup" document (I'm assuming?) I will take note of any more suitable phrase

> With this declaration, Orleans will inject an `ITransactionalState` instance with a state loaded from the transactional state storage named "TransactionStore" (see Setup).